arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -1006,11 +1028,17 @@
           }
         ],
         "responses": {
+          "200": {
```

Comment:
@jhendrixMSFT, Yes here and in other places, we are just correcting the swagger.  The service has been returning the 200, the swagger was not correct for the older specs